docs/en/Authorization.md

```diff
@@ -236,6 +236,25 @@ When you write this code inside your permission definition provider, it finds th
 
 > Tip: It is better to check the value returned by the `GetPermissionOrNull` method since it may return null if the given permission was not defined.
 
+### Permission depending on a Condition
+
+Permission states can be managed according to a condition such as a Feature, Global Feature, or a custom condition. Any condition which implements `IPermissionStateProvider` can be added via calling `AddStateProvider()` method for **PermissionDefinition**.
```

Comment:
It is not clear what is "Permission states". We should explain it like, "You may want to disable a permission based on a condition. Disabled permissions are not visible on the UI and always returns `prohibited` when you check them...". We should write that in the beginning to make the reader to understand what they will achieve when they learn this feature.